lib/departure.rb

```diff
@@ -35,12 +35,7 @@ class << self
         alias original_migrate migrate
       end
 
-      # Checks whether arguments are being passed through PERCONA_ARGS when running
-      # the db:migrate rake task
-      #
-      # @raise [ArgumentsNotSupported] if PERCONA_ARGS has any value
       def migrate(migrations_paths, target_version = nil, &block)
-        raise ArgumentsNotSupported if ENV['PERCONA_ARGS'].present?
         original_migrate(migrations_paths, target_version, &block)
       end
     end
```

Comment:
We should be able to remove this whole `instance_eval` block now, since we are not adding any extra behaviour to this method any more

Reply:
that's a good point and it will make me happy! less metaprogramming :heart_eyes: